abdulfataiaka/pluralsight-admin
# #161290210 Setup for production

#### What does this PR do
- Sets up application for deployment

#### Description of work done
- Refactor webpack config
- Refactor redux store creator

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to the branch `ft-create-prod-build-161290210`
- Check for webpack configuration for production

#### Relevant pivotal tracker story
[161290210](https://www.pivotaltracker.com/story/show/161290210)
